ql/src/java/org/apache/hadoop/hive/ql/plan/FileSinkDesc.java

```diff
@@ -106,7 +108,7 @@
 
   private Set<FileStatus> filesToFetch = null;
 
-  private Set<String> dynPartitionValues = new HashSet<>();
+  private Map<String, List<Path>> dynPartitionValues = new HashMap<>();
```

Comment:
nit: a comment would be nice describing keys and values

Reply:
done